packages/app/firebase-schema.json

```
@@ -81,6 +81,10 @@
         "android_task_executor_keep_alive_seconds": {
           "description": "Keep-alive time of ThreadPoolExecutor used by RNFirebase for Android, in seconds. Defaults to `3`.\n Excess threads in the pool executor will be terminated if they have been idle for more than the keep-alive time.",
           "type": "number"
+        },
+        "background_activity_names": {
+          "description": "On Android, list the names of additional native activities which are used outside the context of the react native application",
```

suggest add statement of purpose
```suggestion
          "description": "The names of Activities used outside the context of react native, to be ignored when determining if foreground or background javascript handlers should be called",
```